src/plugins/github/declaration.js

```diff
@@ -99,35 +99,31 @@ const mentionsAuthorEdgeType = Object.freeze({
   forwardName: "mentions author of",
   backwardName: "has author mentioned by",
   defaultForwardWeight: 1,
-  // TODO(#811): Probably change this to 0
-  defaultBackwardWeight: 1 / 32,
+  defaultBackwardWeight: 0,
```

Comment:
Please put these in a separate change.

Reply:
Note: if they are in a separate change, then the UI is briefly in an inconsistent state, because the sliders start at 1/32, (with a reasonable-looking visual position), but once the slider is moved, it can never return to 1/32. I suppose it's not too objectionable, but that was why I suggested just putting the weight value change in this commit.

Reply:
Ah, good point. Either way is fine with me, then.